docs/usage/self-hosted-configuration.md

```diff
@@ -370,7 +370,7 @@ To handle the case where the underlying Git processes appear to hang, configure
 ## gitUrl
 
 Override the default resolution for Git remote, e.g. to switch GitLab from HTTPS to SSH-based.
-Currently works for GitLab only.
+Currently works for Bitbucket Server and GitLab only.
```

Comment:
```suggestion
```

Add `supportedPlatform` to its config options definition instead

Reply:
Is there another config option to update besides https://github.com/renovatebot/renovate/pull/13875/commits/affda143098433a19f6d249d01ac0cf21528fd3c ?

Reply:
No other